packages/vscode/src/extension.ts

```diff
@@ -203,30 +207,30 @@ export async function activate(context: ExtensionContext): Promise<void> {
       client = createLanguageServer(serverOptions, clientOptions)
       context.subscriptions.push(client.start())
       await client.onReady()
-      window.showInformationMessage('Prisma language server restarted.')
+      await window.showInformationMessage('Prisma language server restarted.')
```

Comment:
We don't want to block on the UI message, do we?

Reply:
True, So just disable the linter here then?

Reply:
I think so. Adding a disable comment + another comment to explain why this lint item is disabled would do.